esmvaltool/utils/recipe_test_workflow/rose-suite.conf

```diff
@@ -0,0 +1,24 @@
+[template variables]
+DRS_ANA4MIPS="default"
+DRS_CMIP3="default"
+DRS_CMIP5="default"
+DRS_CMIP6="default"
+DRS_CORDEX="default"
+DRS_NATIVE6="default"
+DRS_OBS="default"
+DRS_OBS4MIPS="default"
+DRS_OBS6="default"
+ENV_NAME=""
+KGO_ROOT_PATH=""
+MAX_PARALLEL_TASKS=4
```

Comment:
Is this parameter set for all recipes? Because some of them can only run using 1 task at a time.

Reply:
`MAX_PARALLEL_TASKS` can be overridden for a specific recipe; this is the default value.

Reply:
If there is a better default value I would be happy to change it! 😊